### DISCUSSION TITLE
Typo in landing page?

### DISCUSSION THREAD

**Discussion by isaacdarcilla** [alpinejs/alpine#3842] [4. General]
In x-effect section, word `dependancies` is typo error, I think this should be `dependencies`? Even browser auto-correct screams and shows squiggly.

**x-effect**
Execute a script each time one of its dependancies change

In here, https://alpinejs.dev/

**Comment by SimoTod** [comment]
Ironically, there's also a typo in the discussion title. :)

Only the maintainer can update the landing page.
